soh/soh/UIWidgets.cpp

```diff
@@ -333,7 +333,10 @@ namespace UIWidgets {
         }
         if (ImGui::SliderFloat(id, &val, min, max, format))
         {
-            CVar_SetFloat(cvarName, val);
+            if (isPercentage)
+                CVar_SetFloat(cvarName, roundf(val * 100) / 100);
+            else
+                CVar_SetFloat(cvarName, roundf(val * 10) / 10);
```

Comment:
```suggestion
            if (isPercentage) {
                CVar_SetFloat(cvarName, roundf(val * 100) / 100);
            } else {
                CVar_SetFloat(cvarName, val);
            }
```

Reply:
Ah yeah, the other style is used in several places throughout the file so some cleanup should be done there.
Changed it for that function now.

Regarding removing the `roundf(val * 10) / 10)` for non-percentage sliders.
That would no longer round the values for things like `Input Scale` which displays the values with `.1` precision.
Maybe rounding everything to `.01` percentage could work? Do we need more precision for anything?

Reply:
I can't think of any setting that would benefit from more precision than 0.01.